src/API/Google/Merchant.php

```diff
@@ -348,4 +348,20 @@ public function has_access( string $email ): bool {
 	public function update_merchant_id( int $id ): bool {
 		return $this->options->update( OptionsInterface::MERCHANT_ID, $id );
 	}
+
+	/**
+	 * A standalone account is an account created directly in MC without using our plugin.
+	 * Standalone accounts have no access to WCS proxy and other features.
+	 *
+	 * This function detects if the account is standalone
+	 * We use this API for that https://developers.google.com/shopping-content/reference/rest/v2.1/accounts/authinfo
+	 *
+	 * We consider an accou t standalone if no aggregatorId is present or no accountIdentifiers are available.
+	 *
+	 * @return bool True if it's a standalone account.
+	 */
+	public function is_standalone(): bool {
+		$type = $this->service->accounts->authinfo();
+		return ! isset( $type['accountIdentifiers'] ) || empty( $type['accountIdentifiers'] ) || is_null( $type['accountIdentifiers'][0]['aggregatorId'] );
+	}
```

Comment:
I don't think this function is going to catch all the scenarios we need. There are other "manager" accounts which can create sub accounts, we need to specifically detect if our WooCommerce account is the aggregator account.

Since we don't store the ID in the plugin itself we moved the request to get Merchant Accounts to the WCS so we can determine whether it's a sub account or not. See [here](https://github.com/Automattic/woocommerce-connect-server/blob/woogle/lib/google/index.js#L499)

We should consider whether we want to add another end point to check if it's a sub account. Or if we want to cache the isSubAccount value during onboarding.

Also I think the naming might be confusing since in our case `isSubAccount` means it is both a sub account and a child of our WooCommerce account. All other cases are the ones we want to exclude but the word "standalone" doesn't seem to cover sub accounts that have a different parent account.

Reply:
> Since we don't store the ID in the plugin itself we moved the request to get Merchant Accounts to the WCS so we can determine whether it's a sub account or not. See [here](https://github.com/Automattic/woocommerce-connect-server/blob/woogle/lib/google/index.js#L499)
We should consider whether we want to add another end point to check if it's a sub account. 

Thanks for pointing this out.

I modified a bit the approach. Now I use get_merchant_accounts to get if its a subaccount. 

> Or if we want to cache the isSubAccount value during onboarding.

Yes would be cool to create a future PR maybe to cache that endpoint if needed.